qemu/tests/migration_multi_host_downtime_and_speed.py

```diff
@@ -55,7 +56,9 @@ def __init__(self, test, params, env):
             self.speed_step = int((self.max_speed - self.min_speed) /
                                   speed_count)
 
```

Comment:
Hi @suqinhuang 
for linux, using "cpuflags-test/src --stressmem" to create dirty page
for windows, using win_heavyload.